content/tutorials/webrtc/infrastructure/en/index.html

```diff
@@ -511,98 +489,98 @@ <h3 id="more-about-stun">STUN</h3>
+<p>You may also want to take a look at Justin Uberti's proposed IETF standard for a <a href="https://tools.ietf.org/html/draft-uberti-rtcweb-turn-rest-00" title="IETF draft: A REST API For Access To TURN Services">REST API for access to TURN Services</a>.</p>
 
-<p>It's easy to imagine use cases for media streaming that go beyond a simple one-to-one call: for example, video conferencing between a group of colleagues, or a public event with one speaker and hundreds (or millions) of viewers.</p>
+<p>It's easy to imagine use cases for media streaming that go beyond a simple one-to-one call. For example, video conferencing between a group of colleagues or a public event with one speaker and hundreds or millions of viewers.</p>
 
-<p>A WebRTC app can use multiple RTCPeerConnections so to that every endpoint connects to every other endpoint in a mesh configuration. This is the approach taken by apps such as <a href="http://talky.io">talky.io</a>, and works remarkably well for a small handful of peers. Beyond that, processing and bandwidth consumption becomes excessive, especially for mobile clients.</p>
+<p>A WebRTC app can use multiple RTCPeerConnections so that every endpoint connects to every other endpoint in a mesh configuration. This is the approach taken by apps, such as <a href="https://talky.io">talky.io</a>, and works remarkably well for a small handful of peers. Beyond that, processing and bandwidth consumption becomes excessive, especially for mobile clients.</p>
 
 <figure>
   <img src="mesh.png" alt="Mesh: small N-way call" />
-  <figcaption>Full mesh topology: everyone connected to everyone</figcaption>
+  <figcaption>Full mesh topology: Everyone connected to everyone</figcaption>
 </figure>
 
-<p>Alternatively, a WebRTC app could choose one endpoint to distribute streams to all others, in a star configuration. It would also be possible to run a WebRTC endpoint on a server and construct your own redistribution mechanism (a <a href="https://code.google.com/p/webrtc/source/browse/#svn%2Ftrunk%2Ftalk">sample client application</a> is provided by webrtc.org).</p>
+<p>Alternatively, a WebRTC app could choose one endpoint to distribute streams to all others in a star configuration. It would also be possible to run a WebRTC endpoint on a server and construct your own redistribution mechanism (a <a href="https://code.google.com/p/webrtc/source/browse/#svn%2Ftrunk%2Ftalk">sample client app</a> is provided by webrtc.org).</p>
 
-<p>Since Chrome 31 and Opera 18, a MediaStream from one RTCPeerConnection can be used as the input for another: there's a demo at <a href="http://simpl.info/rtcpeerconnection/multi">simpl.info/multi</a>. This can enable more flexible architectures, since it enables a web app to handle call routing by choosing which other peer to connect to.</p>
```

Comment:
Please reinstate the demo using this link: https://webrtc.github.io/samples/src/content/peerconnection/multiple/

Reply:
Will fix